README.md

```diff
@@ -28,40 +28,34 @@ PhoneFormField();
 
 // all params
 PhoneFormField(
-  key: inputKey,
-  controller: controller,
-  autofocus: true,
-  autofillHints: [AutofillHints.telephoneNumber],
-  validator: PhoneValidator.compose([
-    // list of validators to use (see Built-in validators section below)
-    PhoneValidator.required(errorText: "You must enter a value"),
-    PhoneValidator.invalidMobile(),
-    // ...
-  ]),
-  selectorNavigator: const BottomSheetNavigator(),
-  defaultCountry: 'FR',
+  controller: null,     // controller & initialValue value

```

Comment:
I would keep the `key` property as comment above states that all params are visible here ;)